src/main/java/com/agricraft/agricore/json/AgriSerializable.java

```diff
@@ -12,4 +12,6 @@
 
     void setPath(String path);
 
+    String getVersion();
```

Comment:
This might be interesting if the AgriSerializables are nested, but I can't remember if they ever are.

Reply:
Interesting in that you'd see the version number twice.

Reply:
They are not, there are nested classes, but none of the classes being nested are AgriSerializables

Reply:
Ok good.